src/emulator/gui/include/gui/state.h

```diff
@@ -93,6 +93,7 @@ struct State {
     char disassembly_count[5] = "100";
     std::vector<std::string> disassembly;
 
+    std::string lle_module;
```

Comment:
Why is this here? Just make it a local in `draw_settings_dialog`.

Reply:
Is @1whatleytay have said me need set here, for get constant value and no lost input set

Reply:
i have founded how get constant value inside settings dialog